src/instance/defaults.js

```diff
@@ -134,13 +131,5 @@ const defaults = {
     return defaultInteractions().extend(farmMapDefaults);
   },
 
-  // Behaviors.
-  // These are custom sets of map behavior logic provided by farmOS-map in the
-  // src/behavior directory.
-  behaviors() {
-    return [
-      rememberLayer,
-    ];
-  },
```

Comment:
These changes eliminate `defaults.behaviors()` in favor of putting the `rememberLayer` behavior in `farmOS.map.behaviors` which are also loaded by default.

This opens up the possibility of a page customizing `farmOS.map.behaviors` to drop the `rememberLayer` behavior and makes things more consistent by only having one source of automatically loaded behaviors.